src/constants/gl_markers.js

```diff
@@ -17,5 +17,40 @@ module.exports = {
     diamond: '◆',
     'diamond-open': '◇',
     cross: '+',
-    x: '❌'
+    x: '❌',
+    'triangle-up': '▲',
+    'triangle-up-open': '△',
+    'triangle-down': '▼',
+    'triangle-down-open': '▽',
+    'triangle-left': '◄',
+    'triangle-left-open': '◁',
+    'triangle-right': '►',
+    'triangle-right-open': '▷',
+    'triangle-ne': '◥',
+    'triangle-ne-open': '◹',
+    'triangle-nw': '◤',
+    'triangle-nw-open': '◸',
+    'triangle-se': '◢',
+    'triangle-se-open': '◿',
+    'triangle-sw': '◣',
+    'triangle-sw-open': '◺',
+    'pentagon': '⬟',
+    'pentagon-open': '⬠',
+    'hexagon': '⬢',
+    'hexagon-open': '⬡',
+    'hexagon2': '⬣',
+    'star': '★',
+    'star-open': '☆',
+    'hexagram': '✶',
+    'diamond-tall': '♦',
+    'diamond-tall-open': '♢',
+    'hourglass': '⧗',
+    'hourglass-open': '⧖',
+    'bowtie': '⧓',
+    'bowtie-open': '⋈',
+    'circle-cross': '⊕',
+    'circle-x': '⊗',
+    'square-cross': '⊞',
+    'square-x': '⊠',
+    'asterisk': '✱'
```

Comment:
There are two other tweaks this is going to need before it can really match how the svg symbols behave:
- all the symbols should have roughly the same scale. You can see from the svg test image that this is a little bit subjective, for example the side of the square is the same as the diameter of the circle, but the outer width of X and + are a little bigger - basically I tweaked each one until put on a plot next to the other I thought they all looked "about the same size"
- the symbol should be placed with its centroid at the data location, not the center of the bounding rect. For most of the symbols these are the same, but for the triangles they are not (and I guess also for the pentagons and 5-sided stars, to a lesser extent).